src/content/docs/infrastructure/host-integrations/host-integrations-list/redmine-integration.mdx

```diff
@@ -0,0 +1,63 @@
+
+Once installed, confirm that New Relic is ingesting your Redmine data by going to **[one.newrelic.com](https://one.newrelic.com/all-capaibilities)**, then clicking **Metrics & events**. Run the following NRQL query to confirm that New Relic is receiving Redmine data:
+
+```sql
+SELECT apm.service.transaction.duration FROM Metric
+```
+
+## Install the infrastructure agent [#infrastructure]
+
+While our Redmine integration isn't built with our infrastructure agent, we recommend installing it if you want performance data about your hosts. You can install the infrastructure agent two different ways: 
+	
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview). 
+
+* If you’d rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/).
+
+## See Redmine metrics in a dashboard [#dash]
+
+Our default dashboards transform that raw data into charts and graphs, which provide a bird’s eye view of your system’s health. To install our default dashboards, go to our [Redmine instant observability page](https://newrelic.com/instant-observability/redmine). 
+
+## Create queries for customized dashboards [#custom]
+
+Certain performance metrics aren't available out of the box with our pre-built dashboards. While optional, we recommend additional instrumentation so you can track Redmine metrics such as manageable, unmanageable, and overdue issues. To start tracking, install the [Monitoring & Controlling](https://www.redmine.org/plugins/monitoring-controlling) plugin for your Redmine application.
+
+1. Go to your `redmine-monitoring-controlling` folder and open the file named `home_monitoring_controlling_project_controller.rb`. This is found in your plugins directory.
+2. Insert this snippet at the bottom of the file: `::NewRelic::Agent.record_metric(‘Custom/managementIssues’,@managementissues.count)`
+3. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, then click **Metrics & events**. Run the following query to confirm that New Relic is reporting custom metrics:
+
+```sql
+SELECT newrelic.timeslice.value FROM Metric WHERE metricTimesliceName like '%Custom/managementissues%' AND appName = “<your application name>”
+```
+
```

Comment:
## Get Redmine metrics as a dashboard
Once you've installed the [Redmine quickstart](https://newrelic.com/instant-observability/redmine), you can see your critical Redmine data in New Relic. To find your dashboard in New Relic: Go to [one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards, then select Redmine. You can now query your data. For example:

```
SELECT average(databaseDuration) as 'seconds' from Transaction timeseries
```